src/XeroPHP/Application/PartnerApplication.php

```diff
@@ -8,8 +8,8 @@ class PartnerApplication extends Application
 {
     protected static $_type_config_defaults = [
         'xero' => [
-            'site'     => 'https://api-partner.network.xero.com',
-            'base_url' => 'https://api-partner.network.xero.com',
+            'site'     => 'https://api.xero.com',
+            'base_url' => 'https://api.xero.com',
```

Comment:
Ok - so if I understand correctly - you've already made the update and will ignore my pull request.  Any idea when you'll release 2.0?   All new App Partners are not getting entrust certificates so we're actively updating libraries to support these developers.

An email about this change to all existing App partners is scheduled for Jan 13th